tests/PHPUnit/Framework/TestCase/UnitTestCase.php

```diff
@@ -18,16 +21,45 @@
  */
 abstract class UnitTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
It doesn't "hurt" to have this class. And if it's just for consistency with other classes "IntegrationTestCase" and "SystemTestCase" I'd say.